.vscode/settings.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Is this file intended to be here? I see some customizations, which may not be to everyone's liking.

Reply:
The colors? :) It is required for the eslint config to get it right. Don't know if that config can be "global". I will remove the colors.

Reply:
FYI, The colors are only "the shell" of VScode customizations though, using the Peacock official extension.

Reply:
Removed! no longer needed.